### ISSUE TITLE
Jump to Elixir source definition

### ISSUE THREAD

**Issue by rodrigues** [lexical-lsp/lexical#772]
Hey, thanks for all the amazing work with lexical.

Jumping to erlang core code (e.g. `:timer.minutes(5)`) is working great, getting to asdf erlang source.

I can't jump to elixir core code though (elixir 1.16.3, erlang 26.2.5).

I imagine it's not an issue, it's just not implemented yet, right?

It would be so great to have this.

Cheers!

**Comment by scohen** [comment]
Correct. I’m actually not sure why erlang works but elixir doesn’t, but we don’t index the elixir source. I’ll investigate soon

**Comment by miguno** [comment]
For elixir-ls (another LSP for elixir), the solution to this problem was to install elixir from source. Perhaps this solution applies to this issue, too?

**Comment by rodrigues** [comment]
hi @miguno, thanks for the suggestion.

I've just tried it locally, prepending to `$PATH` in `start_lexical.sh` the elixir compiled from source (1.16.3), it didn't work:

```elixir
[jsonrpc] e[10:56:30.052] --> textDocument/definition[26] {"jsonrpc":"2.0","id":26,"method":"textDocument/definition","params":{"textDocument":{"uri":"file:///.../application.ex"},"position":{"line":14,"character":25}}}
[jsonrpc] e[10:56:30.086]   <-- textDocument/definition[26] {"id":26,"result":null,"jsonrpc":"2.0"}
```

**Comment by JamesLavin** [comment]
Want to +1 @rodrigues's words of appreciation for Lexical and his hope that it will some day allow jumping to Elixir source code.

I used to use Vim + Tmux but have been using VS Code in recent years. Trying now to switch to NeoVim (and escape the mouse!). I found configuring NeoVim+Elixir confusing (esp. since I started with `elixir-ls` before discovering `lexical`). Have it mostly working now but thought my config was partially broken till discovering this thread.

FWIW, I install Erlang and Elixir via `asdf`. I believe `asdf`'s pretty popular, so -- in an ideal world -- Lexical will be able to find `asdf`-installed Elixir source definitions.
